### ISSUE TITLE
New release to fix panic on bad arguments to trim --limit

### ISSUE THREAD

**Issue by JayAndCatchFire** [matthiaskrgr/cargo-cache#102]
I was seeing panics like this when I tried `--limit 0`, `--limit 1` or any single digit number, as well as sillier things like `--limit abc`:
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ParseFloatError { kind: Empty }', src/commands/trim.rs:88:66
stack backtrace:
   0: rust_begin_unwind
             at /rustc/9bc8c42bb2f19e745a63f3445f1ac248fb015e53/library/std/src/panicking.rs:493:5
   1: core::panicking::panic_fmt
             at /rustc/9bc8c42bb2f19e745a63f3445f1ac248fb015e53/library/core/src/panicking.rs:92:14
   2: core::option::expect_none_failed
             at /rustc/9bc8c42bb2f19e745a63f3445f1ac248fb015e53/library/core/src/option.rs:1329:5
   3: core::result::Result<T,E>::unwrap
             at /rustc/9bc8c42bb2f19e745a63f3445f1ac248fb015e53/library/core/src/result.rs:1037:23
   4: cargo_cache::commands::trim::parse_size_limit_to_bytes
             at ./src/commands/trim.rs:88:30
   5: cargo_cache::commands::trim::trim_cache
             at ./src/commands/trim.rs:113:22
   6: cargo_cache::main
             at ./src/main.rs:180:31
   7: core::ops::function::FnOnce::call_once
             at /rustc/9bc8c42bb2f19e745a63f3445f1ac248fb015e53/library/core/src/ops/function.rs:227:5
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```
I was using 0.6.2, but I see this is fixed on master thanks to https://github.com/matthiaskrgr/cargo-cache/commit/729866660855b93813ab6265f550623d82e1a22a.  It would be nice to get an official release that includes this bug fix.

**Comment by JayAndCatchFire** [comment]
It also fixes #99, to which I also fell victim.

**Comment by matthiaskrgr** [comment]
Just released 0.6.3, let me know if there is anything else I can do. :)

**Comment by JayAndCatchFire** [comment]
Perfect! Thank you!
